### ISSUE TITLE
Wrong super methods called in touchesEnded: and touchesCancelled:

### ISSUE THREAD

**Issue by getaaron** [designatednerd/DNSCastroSegmentedControl#6]
```
- (void)touchesEnded:(NSSet *)touches withEvent:(UIEvent *)event
{
    [super touchesMoved:touches withEvent:event];
    [self touchesEndedOrCancelled];
}

- (void)touchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event
{
    [super touchesMoved:touches withEvent:event];
    [self touchesEndedOrCancelled];
}
```

These methods both call `touchesMoved:` instead of their appropriate `super` method.

If this is intentional, can we document the reason?

**Comment by designatednerd** [comment]
Not intentional.

**Comment by designatednerd** [comment]
Fixed in 1.3.0
